plugin/src/main/java/com/github/gmazzo/gocd/BuildWatcherPlugin.java

```diff
@@ -237,11 +237,11 @@ private String getChangesResume(PipelineInstance instance) {
                 .map($ -> $.revision + ":\n" + $.comment + " - " + $.userName)
                 .collect(Collectors.joining("\n\n"));
 
-       String result1[] = result.split("\n\n");
+       String result1[] = result.split("[\n.]+");
```

Comment:
If the commit message is like this `Foo. Bar. Something` then it will split by this also no?

Reply:
No it will just split with `\n` here I am checking if there is any subsequent character for example`\n\n`